app/scripts/views/sign_in_recovery_code.js

```diff
@@ -27,8 +28,10 @@ const View = FormView.extend({
 
     return account.consumeRecoveryCode(code)
       .then((result) => {
-        if (result.remaining < 1) {
-          // TODO Lets handle automatically generating recovery codes separately
+        if (result.remaining < 2) {
```

Comment:
same as https://github.com/mozilla/fxa-content-server/pull/6482/files#r212705977 would it be easier to find this magic number as a constant or in the top of this file ?